react/components/UI/Modal/Modal.js

```diff
@@ -15,6 +15,7 @@ const ModalBackdrop = styled.div`
   left: 0;
   background-color: rgba(255, 255, 255, 0.9);
   z-index: 6001;
+  transform: translate3d(0, 0, 1px);
```

Comment:
?

Reply:
If you're trying to force GPU then use `transform: translatez(0);` —this is almost certainly going to cause aliasing bugs.

Reply:
Moves it up one pixel on the z-axis

Reply:
Also why

Reply:
I know what it does but that doesn't explain anything.

Reply:
We do this on the header bar to fix position:fixed bugs on mobile safari

Reply:
Did you actually experience an issue on Mobile Safari? Can you document why it needs to be 1px in a comment pointing to some external source? Can we move it into a media query that only executes on mobile devices.